build.gradle

```diff
@@ -400,6 +404,87 @@ subprojects { subproj ->
 		}
 	}
 
+	// @sbrannen, @marcphillip:
+	// Each new release of error-prone brings with it new default checks.
+	// It seems possible to explicitly tell error-prone _which_ checks we want to enable and
+	// disable, similarly to how we specify "-Xlint" flags above.
+	// Do we want to customise exactly which checks error-prone should run? Or are we happy to just
+	// accept the default checks that each new release brings, explictly excluding the ones we dislike?
+	if (JavaVersion.current() == JavaVersion.VERSION_1_8 && subproj.name != 'documentation') {
```

Comment:
Great, thanks for clarifying things for me @sbrannen! I'll work on reverting this comment now.